brainstormforce/astra
# AST-2542 - fix: remove duplicated footer css

### Description

- Footer Widgets Dynamic CSS gets duplicated.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2542

### Screenshots

https://share.bsf.io/DOukWjr0
### Types of changes


- Call to `Astra_Widget_Component_Dynamic_CSS::astra_widget_dynamic_css( 'footer' )` was being invoked inside loop iterating over no of widgets, but `Astra_Widget_Component_Dynamic_CSS::astra_widget_dynamic_css( 'footer' )` already has its own loop to apply the necessary footer widgets css.
- This was resulting in duplicated css being appended to footer widgets.
- Moved the method invocation `Astra_Widget_Component_Dynamic_CSS::astra_widget_dynamic_css( 'footer' )` outside the loop.



### How has this been tested?

- In Footer Builder, add a widget.
- Choose a Quote block for example, and add some design styles.
- View in frontend, inspect the element & observe the `.footer-widget-area[data-section="sidebar-widgets-footer-widget-2"]` CSS rule, it should not be duplicated.

### Checklist:
